playbooks/roles/ocp-service-binding-operator/tasks/main.yml

```diff
@@ -48,6 +48,14 @@
 - name: Check all nodes are healthy
   shell: oc wait --all --for=condition=Ready nodes 
 
+# Get the cluster version
+- name: Get the cluster version
+  shell: oc version | awk 'NR==1{print $3}' | cut -d'.' -f1-2
```

Comment:
This will give you the `oc` client version. Use `grep Server` to consume cluster version.

Reply:
OK Yussufsh let me check and update on it